lab-cop/config/definitions/lab-cop.yml

```diff
@@ -0,0 +1,41 @@
+- scm:
+    name: lab-cop
+    scm:
+      - git:
+          url: https://github.com/ceph/teuthology.git
+          branches:
+            - ${sha1}
```

Comment:
this is usually done for pull requests, are you looking to trigger this on PRs? if so you will need a `triggers:` section with that defined in. Otherwise you can trigger on merges, in which case you need to remove the `branches` section (it defaults to anything/everything) *or* define what branches you are interested in